e2e/test/scenarios/custom-column/custom-column.cy.spec.js

```diff
@@ -633,25 +639,18 @@ describe("scenarios > question > custom column", () => {
       .should("be.visible");
   });
 
-  it("should allow switching focus with Tab", () => {
+  it("should allow indenting using Tab", () => {
     H.openOrdersTable({ mode: "notebook" });
     cy.findByLabelText("Custom column").click();
 
     H.enterCustomColumnDetails({ formula: "1 + 2", blur: false });
 
-    // next focus: the textbox for the name
+    // Tab should insert indentation
     cy.realPress("Tab");
-    cy.focused().should("have.attr", "value").and("eq", "");
-    cy.focused()
-      .should("have.attr", "placeholder")
-      .and("eq", "Something nice and descriptive");
-
-    // Shift+Tab and we're back at the editor
-    cy.realPress(["Shift", "Tab"]);
-    cy.focused().should("have.attr", "class").and("eq", "cm-content");
+    H.CustomExpressionEditor.value().should("equal", "1 + 2  ");
   });
 
-  it("should allow tabbing away from, then back to editor, while formatting expression and placing caret after reformatted expression", () => {
+  it("should not format expression when tabbing away from the editor", () => {
```

Comment:
It's not possible to tab away

```suggestion
  it("should not format expression when tabbing in the editor", () => {
```